url-gen/src/main/kotlin/com/cloudinary/transformation/delivery/DeliveryActions.kt

```diff
@@ -44,17 +44,22 @@ class DeliveryFormat(
     private val format: Format,
     private val lossy: Boolean? = null,
     private val progressive: Progressive? = null,
-    private val preserveTransparency: Boolean? = null
+    private val preserveTransparency: Boolean? = null,
+    private var ignoreMaskChannels: Boolean = false
```

Comment:
Why the default value is `false` opposed to `null` in the other variables?

Reply:
Fixed